tests/EdgeToEdge/Routes/DisplayPageRouteTest.php

```diff
@@ -19,21 +19,19 @@
 class DisplayPageRouteTest extends WebRouteTestCase {
 
 	public function testWhenPageHasCustomTemplate_customTemplateIsRendered(): void {
-		$this->createEnvironment(
-			[],
-			function ( Client $client, FunFunFactory $factory ): void {
-				$factory->setContentPagePageSelector( $this->newMockPageSelector( 'test' ) );
-				$factory->setContentProvider( $this->newVfsContentProvider( [ 'test.twig' => '' ] ) );
-
-				$crawler = $client->request( 'GET', '/page/test' );
-
-				$this->assertCount( 1, $crawler->filter( '.test-block' ) );
-			}
+		$client = $this->createClient( [ 'skin' => [ 'default' => 'cat17' ] ] );
+		//supporters.twig page from fundraising-frontend-content repository
```

Comment:
This comment is a bit misleading, as the fundraising-frontend-content  is not really relevant here. I think you meant `page_layouts/supporters.html.twig`